src/main/resources/assets/xpbook/lang/ru_ru.json

```diff
@@ -0,0 +1,6 @@
+{
+    "item.xpbook.xp_book": "Том опыта",
+    "xpbook.tooltip.1": "Shift + пкм, чтобы сохранить такое же кол-во Опыта насколько это возможно",
```

Comment:
`Shift` is misleading here, as people can rebind their sneaking key to anything.